test/winter_onboarding_2021/fleet_management/db/fleet_test.clj

```diff
@@ -0,0 +1,21 @@
+(ns winter-onboarding-2021.fleet-management.db.fleet-test
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [winter-onboarding-2021.fleet-management-service.db.fleet :as fleet-db]
+            [winter-onboarding-2021.fleet-management-service.db.user :as user-db]
+            [winter-onboarding-2021.fleet-management.fixtures :as fixtures]))
+
+(use-fixtures :once fixtures/config fixtures/db-connection)
+(use-fixtures :each fixtures/clear-db)
+
+(deftest create-fleet
```

Comment:
Should this test also do a lookup to test that the data is actually written to the db?

Reply:
Yes, thats how we will know that, the fleet got created successfully isn't it?